README.md

```diff
@@ -69,6 +69,21 @@ You can install and run the NativeScript CLI on Windows or OS X.
 
 > On Windows systems, you can develop, build, and deploy NativeScript projects that target Android.
 
+### Setup Script
+To quickly set up your system for the latest NativeScript CLI, paste the following PowerShell script in the **command prompt** and hit `Enter`:
+```cmd
+@powershell -NoProfile -ExecutionPolicy Bypass -Command "iex ((new-object net.webclient).DownloadString('https://raw.githubusercontent.com/NativeScript/nativescript-cli/production/setup/native-script.ps1'))"
+```
+
+Alternatively, your can paste the following PowerShell setup script in a **Windows PowerShell console** and hit `Enter`:
+```PowerShell
+start-process -FilePath PowerShell.exe -Verb Runas -Wait -ArgumentList "-NoProfile -ExecutionPolicy Bypass -Command iex ((new-object net.webclient).DownloadString('https://raw.githubusercontent.com/NativeScript/nativescript-cli/production/setup/native-script.ps1'))"
+```
+> Both scripts require that you have **.NET 4.0 or later** installed on your system.
```

Comment:
The fwlink is a link which Microsoft promised to keep alive. The search link from the script will break sooner or later. At any rate, this is a *suggestion* to the user, if they need another version, they are free to do so.

The idea of the readme is to point the **novice** users in the correct direction with least friction. Expert users know what to do better than us.